.github/workflows/rust.yml

```diff
@@ -16,11 +16,39 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        fls: true
```

Comment:
typo? should it be `lfs`?

Reply:
Ohhhh, that's explains why the LFS files weren't checked out